package.json

```diff
@@ -36,6 +36,7 @@
     "gulp-watch": "^4.3.11",
     "istanbul": "^0.4.5",
     "jsdoc-babel": "^0.2.1",
+    "jsdoc-baseline": "git+https://github.com/sugarcrm/jsdoc-baseline.git",
```

Comment:
@rhoggSugarcrm I tried to use gulp-jsdoc3 in sidecar and doesn't work. I had to use the `@sugarcrm/gulp-jsdoc3` in order to read the templates options
Perhaps I did something wrong but I had to do that here:
https://github.com/sugarcrm/sidecar/blob/SC-master/package.json#L25